windows/Gyroscope.cs

```diff
@@ -35,19 +29,20 @@ private void ReadingChanged(object sender, Sensors.GyrometerReadingChangedEventA
         {
             Sensors.GyrometerReading reading = e.Reading;
 
-            this.SendEvent("Gyroscope", new AccelerometerJsonObject
+            this.SendEvent("Gyroscope", new RNSensorsJsonObject
             {
                 X = reading.AngularVelocityX,
                 Y = reading.AngularVelocityY,
                 Z = reading.AngularVelocityZ,
                 Timestamp = reading.Timestamp
-            });
+            }.ToJObject());
         }
 
         [ReactMethod]
         public void setUpdateInterval(int newInterval)
         {
             this.interval = newInterval;
+            if (_gyrometer != null) _gyrometer.ReportInterval = (uint)this.interval;
```

Comment:
Haha, I've seen it on the android code, but I checked, ReportInterval is in milliseconds ;)